Pipfile

```diff
@@ -21,6 +21,7 @@ regex = "==2020.1.8"
 SQLAlchemy = "==1.4.15"
 toml = "==0.10.0"
 typed-ast = "==1.4.1"
+sqlalchemy-citext = "*"
```

Comment:
I believe we are pinning specific versions of each package even in `Pipfile`. Please determine the actual version installed (run `pip freeze` inside the Pipenv environment, or look in `Pipfile.lock`) and specify it here.